SpatialGDK/Source/SpatialGDK/Public/Utils/LayerInfo.h

```diff
@@ -2,7 +2,14 @@
 
 #pragma once
 
-#include "CoreMinimal.h"
```

Comment:
https://docs.unrealengine.com/en-US/Programming/BuildTools/UnrealBuildTool/IWYU/index.html
this seem to suggest that including CoreMinimal.h in every header is not discouraged, but rather an acceptable convention.
What is the reason for removing it in favor of a long list of includes ?

Reply:
Been told we prefer not to have it in the past, maybe @m-samiec knows

Reply:
Nothing I'm aware of. Including "engine.h" and "core.h" used to be quite common way back when, and was replaced with the minimal variants, perhaps that's the confusion?

Reply:
reverted then 🙂